scripts/cat/cats.py

```diff
@@ -766,7 +766,7 @@ def grief(self, body: bool):
                 if (
                     body
                     and not body_treated
-                    and "rosemary" in game.clan.herb_supply.entire_supply
+                    and game.clan.herb_supply.entire_supply["rosemary"] > 0
```

Comment:
wouldn't:
```suggestion
                    and game.clan.herb_supply.entire_supply["rosemary"]
```
work? Since if the supply is zero, it should return false anyways